addons/hr_timesheet/models/project_project.py

```diff
@@ -211,3 +211,55 @@ def action_project_timesheets(self):
         action = self.env['ir.actions.act_window']._for_xml_id('hr_timesheet.act_hr_timesheet_line_by_project')
         action['display_name'] = _("%(name)s's Timesheets", name=self.name)
         return action
+
+    # ----------------------------
+    #  Project Updates
+    # ----------------------------
+
+    def _get_stat_buttons(self):
+        buttons = super(Project, self)._get_stat_buttons()
+        if not self.allow_timesheets or not self.env.user.has_group("hr_timesheet.group_hr_timesheet_user"):
+            return buttons
+
+        encode_uom = self.env.company.timesheet_encode_uom_id
+        hour_uom = self.env.ref('uom.product_uom_hour')
+        uom_ratio = hour_uom.factor / encode_uom.factor
```

Comment:
Quite minor but I guess you dont really need the ``hour_uom`` variable since its only used in the line below